app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -5,7 +5,7 @@
     },
 
     isPayuChoosen: function(e) {
-      return this.choosenPaymentMethodId(e) == this.payuPaymentMethodId();
+      return this.choosenPaymentMethodId(e) === this.payuPaymentMethodId();
```

Comment:
I recommend adding `jshint` linter to your editor. :-) For example [jsHint Gutter](https://github.com/victorporof/Sublime-JSHint) if you are using **Sublime Text**.